doc/source/timeseries.rst

```diff
@@ -175,12 +175,8 @@ you can pass the ``dayfirst`` flag:
    can't be parsed with the day being first it will be parsed as if
    ``dayfirst`` were False.
 
-.. note::
-   Specifying a ``format`` argument will potentially speed up the conversion
-   considerably and on versions later then 0.13.0 explicitly specifying
-   a format string of '%Y%m%d' takes a faster path still.
```

Comment:
@jreback we removed this part, as it is not a very clear sentence, and also does not seem to be true (https://github.com/pandas-dev/pandas/issues/17410)